static/js/controllers/edit-contact.js

```diff
@@ -17,7 +17,7 @@ var libphonenumber = require('libphonenumber/utils'),
           endkey: [{}],
           reduce: false,
           include_docs: true,
-          timeout: false
+          targetScope: "contacts"
```

Comment:
This should be the root scope as the EditContactCtrl doesn't have a route attached - it's loaded once and then kept as a child of InboxCtrl for every state.